nilearn/plotting/html_surface.py

```diff
@@ -101,15 +166,20 @@ def full_brain_info(volume_img, mesh='fsaverage5', threshold=None,
         symmetric_cmap=symmetric_cmap, vmax=vmax, vmin=vmin)
 
     for hemi, surf_map in surface_maps.items():
-        bg_map = surface.load_surf_data(mesh['sulc_{}'.format(hemi)])
+        if isinstance(bg_map, str) and bg_map == "auto":
```

Comment:
to reproduce:
```python
from nilearn import datasets, plotting

img = datasets.fetch_neurovault_motor_task()["images"][0]
plot = plotting.view_img_on_surf(img, threshold=2, surf_mesh="fsaverage")
```

(the problem also occurs with the default fsaverage5 but it is more obvious with the higher-resolution fsaverage. btw in another pr we could consider choosing a different bg map for low-resolution meshes, because the curvature sign looks better for fsaverage but a bit weird for fsaverage5 and lower IMO)

Reply:
Oof, good catch!

Reply:
> Also if you have time could you try the modified functions with each of the possible choices for the new parameters and check that they work as expected? there may be other bugs we might miss otherwise. Thanks!

I went over all methods exposed by the API and tested them for left and right hemispheres with the neurovault motor task and a curv sign background image and didn't spot any bug. The dimming effect after several plots still exists, bug I suggest we fix it in another PR.